go/interpreter/lfvm/memory.go

```diff
@@ -167,7 +177,13 @@ func (m *Memory) SetWord(offset uint64, value *uint256.Int) error {
 	return nil
 }
 
-func (m *Memory) Set(offset, size uint64, value []byte) error {
+func (m *Memory) Set(offset, size uint64, value []byte, c *context, shouldEnsureCapacity bool) error {
```

Comment:
I am not a fan of the `shouldEnsureCapacity` parameter. It is not very intuitive what this final boolean flag is supposed to be for at each call side. I would prefer it always to be enabled or, if this causes problems, have a dedicated `SetWithoutCapacityCheck(offset, size, value)` .

Reply:
I removed the bool and added `SetWithoutCapacityCheck`. Used similar pattern for `GetSlice`